bundler/task/release.rake

```diff
@@ -65,9 +68,10 @@ namespace :release do
     JSON.parse(response.body)
   end
 
+  desc "Make sure github API is ready to be used"
   task :verify_github do
     require "pp"
-    gh_api_post :path => "/user"
+    PP.pp gh_api :path => "/user"
```

Comment:
Is this `PP.pp`for debugging or intentional?

Reply:
It's intentional, just wanted to give a hint that the authentication went fine, but it's not really necessary since if there's an error, I guess it will be displayed.

Reply:
You're the only person ever using this, so you're free to basically do whatever you want ¯\_(ツ)_/¯. I was just checking

Reply:
Thanks @colby-swandale! This has actually been used for a quite a few releases now, since https://github.com/rubygems/bundler/commit/8e8b8142c98254d42c710f06627624fe0126b104.

Anyways, I'm not really fond of the print and I don't think it adds much value, so I will remove it.

Reply:
I updated the PR to remove the `require` instead. Thanks for the feedback!